src/genie/libs/parser/iosxe/show_lldp.py

```diff
@@ -149,7 +149,7 @@ def cli(self, entry='',output=None):
 
         # initial regexp pattern
         p1 = re.compile(r'^Local +Intf: +(?P<intf>[\w\/\.\-]+)$')
-        p1_1 = re.compile(r'^Port +id: +(?P<port_id>[\w\/\.\-]+)$')
+        p1_1 = re.compile(r'^Port +id: +(?P<port_id>[\S ]+)$')
```

Comment:
to match spaces we typically use ``\s+`` in the regex

Reply:
Hi, thank you for your feedback.

I changed the regex in my last commit as you suggested (forgot to keep the changes after rebasing...)

I also fixed my test:

![test_show_lldp_neighbors_detail_on_iosxev2](https://user-images.githubusercontent.com/32892378/75889663-8d934680-5e2d-11ea-8184-d2e348814d49.jpg)

Reply:
thanks very much! looks good now